src/matches.rs

```diff
@@ -13,8 +13,8 @@
 use std::iter::repeat;
 
 use config::lists::*;
-use syntax::{ast, ptr};
 use syntax::codemap::{BytePos, Span};
+use syntax::{ast, ptr};
```

Comment:
I'm not sure this was intentional or not, but the tests fail without this reordering.